CorsixTH/Lua/entities/patient.lua

```diff
@@ -199,7 +199,7 @@ function Patient:completeDiagnosticStep(room)
     local divisor = math.random(1, 3)
     local attn_detail = room.staff_member.profile.attention_to_detail / divisor
     local skill = room.staff_member.profile.skill / divisor
-    local diagnosis_bonus = (attn_detail + 0.4) *  skill
+    diagnosis_bonus = (attn_detail + 0.4) *  skill
```

Comment:
Two spaces between * and skill. I know you didn't put them there, but since you touched the line anyway, might as well fix it.